tests/bdd/features/volume/capacity_limit/creation.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Volume creation capacity limit
+
+  Background:
+    Given a control plane, Io-Engine instances and a pool
+
+  Scenario: attempted creation exceeding the capacity limit
+    Given a gRPC request to create a volume
+    When the request includes a capacity limit
+    And the volume creation would result in the capacity limit being exceeded
+    Then volume creation should fail with an out-of-range error
```

Comment:
In case we agree on changing error kind mapping.
```suggestion
    Then volume creation should fail with a resource exhausted error
```

Reply:
Will fix with the PR for the tests.